src/vs/workbench/contrib/multiDiffEditor/browser/actions.ts

```diff
@@ -25,9 +26,9 @@ export class GoToFileAction extends Action2 {
 			id: 'multiDiffEditor.goToFile',
 			title: localize2('goToFile', 'Open File'),
 			icon: Codicon.goToFile,
-			precondition: EditorContextKeys.inMultiDiffEditor,
+			precondition: ContextKeyExpr.and(ActiveEditorContext.isEqualTo(MultiDiffEditor.ID), EditorContextKeys.inMultiDiffEditor),
 			menu: {
-				when: EditorContextKeys.inMultiDiffEditor,
+				when: ContextKeyExpr.and(ActiveEditorContext.isEqualTo(MultiDiffEditor.ID), EditorContextKeys.inMultiDiffEditor),
```

Comment:
@hediet 
Based on the code in the `run`, (see line `44 => if (!(activeEditorPane instanceof MultiDiffEditor)) {`)
This action only works for `MultiDiffEditor`, 
Hence having just the condition to check the ActiveEditorContext should suffice, i.e. we do not need `EditorContextKeys.inMultiDiffEditor)`

I need this change, to ensure this menu item isn't displayed in Notebook Diff editor using the multi-file editor widget.

Reply:
Yes, lets remove `inMultiDiffEditor`